src/worker-type.js

```diff
@@ -675,7 +675,8 @@ WorkerType.testLaunchSpecs = function(worker, keyPrefix, provisionerId, provisio
           truePrice: 1,
           region: region,
           type: type,
-          zone: 'fakezone1',
+          // just test the 'a' AZ in each region..
```

Comment:
We should only do this if there are no AZ in the worker type definition, otherwise we should do a test for each availability zone's specializations.